.pre-commit-config.yaml

```diff
@@ -40,3 +40,11 @@ repos:
         args:
           - --exclude-charts=alfresco-common,alfresco-sync-service
           - --values-dir=ci
+  - repo: local
+    hooks:
+    -   id: update-charts-yml
+        name: update charts.yml file
+        language: docker_image
+        entry: ghcr.io/alfresco/build-tools-gen-index-yml:master@sha256:8e077a69e29aff588221c02c4727525ca21c7912569c40122179086b8477f606
+        files: ^charts/[^/]+/Chart.yaml
```

Comment:
seems useless if `pass_filenames` is false

Reply:
it is not, `files` is what triggers the pre-commit to run, while `pass_filenames` is if the changed files needs to be provided as arguments to the invoked `entry`

Reply:
Then I would expect the values files to be part of the triggers as well. Am I misunderstanding how this should work?

Reply:
it's the simplest way I've found to make sure the hook is run whenever a new chart could be potentially been added (git doesn't store directories so a diff can only be applied to a file - Chart.yaml is a file always present in an helm charts - the script is idempotent and fast to run so should not be a problem to run it when it's not really needed)